app/src/components/items/uv/universalViewer.tsx

```diff
@@ -61,79 +70,112 @@ const UniversalViewer: React.FC<UniversalViewerProps> = React.memo(
       if (uv) {
         // override config using an inline json object
         uv.on("configure", function ({ config, cb }) {
```

Comment:
it's unclear which config object is more important. this one, or the one passed down as a prop from view ItemMediaViewer.